phototherapy/js/phototherapy.js

```diff
@@ -21,6 +21,9 @@ app.controller("PhototherapyController", ['$scope', '$rootScope', '$timeout', fu
     var butaniCtx = document.getElementById('butaniChart').getContext('2d');
     var phototherapyCtx = document.getElementById('phototherapyChart').getContext('2d');
 
+    const termsSignedKey = 'mashlom.termsSigned';
+    ctrl.termsSigned = localStorage.getItem(termsSignedKey) === 'true';
```

Comment:
I think that storing timestamp is better (it just gives us more data)
i.e.

mashlom.termsSigned = 123456789
instead of 
mashlom.termsSigned = true